### DISCUSSION TITLE
what does LCD stand for?

### DISCUSSION THREAD

**Discussion by waanito** [artisan-roaster-scope/artisan#1117] [Q&A] [answered]
I've searched the documentation but can't find what LCD stands for. 
It's mentioned in numerous places, e.g., Phases LCDs,  but as far as I can tell never defined. 
I'm fairly sure it doesn't stand for liquid crystal display...
thanks.

**Comment by poundy** [comment]
it does stand for liquid crystal display. 
The characters used on the app represents what you would ordinarily see on LCD displays, and it's just a common name used in many places that are no longer true LCDs but look like it.

**Comment by MAKOMO** [comment, marked as answer]
The full term (but too long on screen) would be [Seven Segment LCD](https://en.wikipedia.org/wiki/Seven-segment_display). Those electric components were originally build with LCD or LED technology. 

The initial script that eventually turned into Artisan v0.0 in 2009 used this term already, which was directly derived from the name of the [UI widget](https://doc.qt.io/qt-6/qlcdnumber.html) used in the implementation. We never gave that a second thought.

**Reply by waanito** [reply]
Thanks for the history. Yes I'm very familiar with 7 segment digit displays on alarm clocks, calculators, etc.
 LCD, to me conjures up a complete display unit that has at least one video port and power, YMMV. 
OTOH I can see the utility of using LCD in the docs as a term for display of some output.  Thanks for clearing up my confusion.
